sample/container.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: build.projectriff.io/v1alpha1
+kind: Container
+metadata:
+  name: upper
+spec:
+  image: _
```

Comment:
I realize it's not new to this, but looking at in context, this feels unintuitive if someone isn't aware of the defaulting, and it seems almost like it shouldn't be required and that should simply be the default if left empty in the spec. Also the comment in the container_types code states:

>  May contain a leading underscore to have the default image prefix applied.

That only refers to "leading" and "prefix", not the defaulting of the image name itself.

Reply:
I updated the comment and apply `_` to empty images in the defaulter. For these resource, I'd still favor being explicit about the image. The previous values were hard coded images in my gcr bucket that nobody else could pull, so `_` is a huge improvement.